lib/cgo/cipher.address.go

```diff
@@ -1,145 +1,323 @@
 package main
 
+import (
+	"reflect"
+	"unsafe"
+
+	cipher "github.com/skycoin/skycoin/src/cipher"
+)
+
 /*
+
 #include <string.h>
 #include <stdlib.h>
-
 #include "skytypes.h"
 
+  #include "skytypes.h"
 */
 import "C"
 
-import (
-	"reflect"
-	"unsafe"
-
-	"github.com/skycoin/skycoin/src/cipher"
-)
-
-/**
- * Functions in github.com/skycoin/skycoin/src/cipher/address.go
- */
-
 //export SKY_cipher_DecodeBase58Address
-func SKY_cipher_DecodeBase58Address(_addr string, _arg1 *C.cipher__Address) uint32 {
+func SKY_cipher_DecodeBase58Address(_addr string, _arg1 *C.cipher__Address) (____error_code uint32) {
+	____error_code = SKY_OK
+	defer func() {
+		____error_code = catchApiPanic(____error_code, recover())
+	}()
+	checkAPIReady()
+
 	addr, err := cipher.DecodeBase58Address(_addr)
-	errcode := libErrorCode(err)
+	____error_code = libErrorCode(err)
 	if err == nil {
 		*_arg1 = *(*C.cipher__Address)(unsafe.Pointer(&addr))
 	}
-	return errcode
+	return
+}
+
+//export SKY_cipher_MustDecodeBase58Address
+func SKY_cipher_MustDecodeBase58Address(_addr string, _arg1 *C.cipher__Address) (____error_code uint32) {
```

Comment:
@stdevHan pls remove all functions with `Must` prefix if they only invoke another API function and panic.